gravitee-rest-api-model/src/main/java/io/gravitee/rest/api/model/PageEntity.java

```diff
@@ -44,8 +41,8 @@
     private boolean homepage;
     private String parentId;
     private String parentPath;
-    @JsonProperty("excluded_groups")
-    private List<String> excludedGroups;
+    private boolean excludedAccessControls;
```

Comment:
```suggestion
    @JsonProperty("excluded_access_controls")
    private boolean excludedAccessControls;
```

Reply:
Why?  In this case, we should change all properties parent_id, parent_path... I'm from js world, I like camelCase